### PR TITLE
Add a Leaflet control to toggle the 3 layers. Closes #33

### LINKED ISSUE #33: Add layer controls to the Test Browser
When the test browser loads a test, all of the different files are displayed on the map. With adequate styling, many of these layers can be viewed at the same time, but that will often be too much at once.

Buttons should be added to the map to toggle the visibility of loaded layers (or switch between preset combos or something).

### PR DESCRIPTION
**Pull request by dabreegster**
Kind of awesome how this is a one-liner. I don't want to say how many layer-like UI controls I've very painstakingly implemented in A/B Street, or how much code they take...

https://user-images.githubusercontent.com/1664407/180495072-79174d29-c925-4373-9fb6-c8af3a6ca5de.mp4

### CODE REVIEW

**Comment by dabreegster** [comment]
Er, my screencapture chopped off the right edge, but rest assured the standard layer control is there

File: tests-web/assets/js/tests.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const makeOpenTest = (map) => async (name) => {
     const networkLayer = await makeDotLayer(await network, { bounds })
     map.addLayer(networkLayer);
 
+    L.control.layers({}, {"Geometry": rawMapLayer, "OSM input": inputLayer, "Graph": networkLayer}).addTo(map);
```
Comment (dabreegster) [inline tests-web/assets/js/tests.js:21]: What're we using for formatting? I used `prettier`, but it changed a bunch of stuff here. VS code had a slight change in the imports. I'm happy to match whatever

Comment (BudgieInWA) [reply tests-web/assets/js/tests.js:21]: Prettier is my preferred formatter (as long as trailing commas is turned on). The existing code does not match the `prettier` format, but I am in favour of changing over to it.

For now, format it however looks decent, and I will set up prettier with a config and reformat everything soon.

**Comment by BudgieInWA** [comment]
> Kind of awesome how this is a one-liner. I don't want to say how many layer-like UI controls I've very painstakingly implemented in A/B Street, or how much code they take...

Re-using UI is one of the biggest draws of web-tech. It's nice that leaflet has lots of our use-cases covered :D

**Pull request merged**